README.md

```diff
@@ -398,6 +399,30 @@ test('passes when value is a function', () => {
 });
 ```
 
+#### .toThrowWithMessage(type, message)
+
+Use `.toThrowWithMessage` when checking if a callback function throws an error with a given error type and given error message.
```

Comment:
Perhaps it's worth mentioning that the message can be either a `String` or `RegExp`?